core_weekly/template.py

```diff
@@ -46,6 +50,26 @@ def headers(self):
 ## A quick update about PrestaShop\'s GitHub issues and pull requests:
 '''
 
+        return template.format(
+            today=today,
+            week=str(week),
+            month=month,
+            year=str(year),
+            first_day=self.format_day_number(first_day_number),
+            last_day=self.format_day_number(last_day_number)
+        )
+
+    def format_day_number(self, day_number):
+
+        if (day_number == '1'):
+            return '1st'
+        elif (day_number == '2'):
+            return '2nd'
+        elif (day_number == '3'):
+            return '3rd'
+        else:
+            return day_number + 'th'
+
```

Comment:
```suggestion
        day_numbers = {
          1: '1st',
          2: '2nd',
          3: '3rd',
        }
        return day_numbers.get(day_number, day_number + 'th')
```

You should be able to do something like that otherwise :)

Reply:
Ooooh python got some nice tricks up its sleeve !

Reply:
Python is great!